chart/templates/deployment.yaml

```diff
@@ -113,6 +113,8 @@ spec:
               value: "{{ .Values.distributor.projectFilter }}"
             - name: SERVICE_FILTER
               value: "{{ .Values.distributor.serviceFilter }}"
+            - name: DISTRIBUTOR_VERSION
```

Comment:
Why do we pass the distributor version to the distributor itself ? Shouldn't it be able to know which version is it ?

Reply:
This is coming as a requirement from Keptn core, not related to my PR.